xray/segment.go

```diff
@@ -384,7 +418,6 @@ func (sub *Segment) beforeEmitSubsegment(seg *Segment) {
 	sub.ParentID = seg.ID
 	sub.Type = "subsegment"
 	sub.RequestWasTraced = seg.RequestWasTraced
-	sub.parent = nil
```

Comment:
Why we remove this line? Does that mean the subsegment after completing streaming strategy will still have a parent?

Reply:
Replied in previous comment, since its more appropriate to explain there.